packages/services/hmi-server/docker/jvm/Dockerfile.jvm

```diff
@@ -1,37 +1,93 @@
 ####
-# Compile and Build the Server using regular JVM
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
```

Comment:
we can remove all this boilerplate comments

Reply:
done